src/tests/topp/ProteomicsLFQ_5_out.mzTab

```diff
@@ -88,20 +88,20 @@ PRT	Q14525|KT33B_HUMAN	Keratin, type I cuticular Ha3-II - Homo sapiens (Human).
-PRT	P02769|ALBU_BOVIN	Serum albumin - Bos taurus (Bovine).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.105263157894737	P02769|ALBU_BOVIN	590-UNIMOD:4	0.401976935749588	1.80142464e08	1.39633376e08	7.4680016e07	1.80142464e08	null	null	1.39633376e08	null	null	7.4680016e07	null	null	null	null	null	single_protein
+PRT	P00761|TRYP_PIG	Trypsin - Sus scrofa (Pig).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.105263157894737	P00761|TRYP_PIG	null	0.077922077922078	0.0	2.0622054e07	0.0	0.0	null	null	2.0622054e07	null	null	0.0	null	null	null	null	null	single_protein
+PRT	P02769|ALBU_BOVIN	Serum albumin - Bos taurus (Bovine).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.105263157894737	P02769|ALBU_BOVIN	590-UNIMOD:4	0.401976935749588	9.0635128e07	1.5465968e08	8.830588799999999e07	9.0635128e07	null	null	1.5465968e08	null	null	8.830588799999999e07	null	null	null	null	null	single_protein
 PRT	P02769|ALBU_BOVIN_rev	Serum albumin - Bos taurus (Bovine).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.105263157894737	P02769|ALBU_BOVIN_rev	296-UNIMOD:4	0.014827018121911	1.35815075e06	0.0	0.0	1.35815075e06	null	null	0.0	null	null	0.0	null	null	null	null	null	single_protein
 PRT	P06871|TRY1_CANFA	Cationic trypsin - Canis familiaris (Dog).	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.105263157894737	P06871|TRY1_CANFA	null	0.040650406504065	null	null	null	null	null	null	null	null	null	null	null	null	null	null	null	single_protein
-PRT	sp|O46375|TTHY_BOVIN	Transthyretin OS=Bos taurus GN=TTR PE=1 SV=1	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.105263157894737	sp|O46375|TTHY_BOVIN	null	0.047619047619048	0.0	3.6438005e06	2.904332e06	0.0	null	null	3.6438005e06	null	null	2.904332e06	null	null	null	null	null	single_protein
-PRT	tr|A9EY18|A9EY18_SORC5	Putative ferredoxin, 2Fe-2S OS=Sorangium cellulosum (strain So ce56) GN=sce7334 PE=4 SV=1	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.105263157894737	tr|A9EY18|A9EY18_SORC5	null	0.084745762711864	0.0	2788.036376953125	0.0	0.0	null	null	2788.036376953125	null	null	0.0	null	null	null	null	null	single_protein
-PRT	tr|A9G0B6|A9G0B6_SORC5	Putative uncharacterized protein OS=Sorangium cellulosum (strain So ce56) GN=sce8672 PE=4 SV=1	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.105263157894737	tr|A9G0B6|A9G0B6_SORC5	null	0.112676056338028	7877.7080078125	6914.94091796875	0.0	7877.7080078125	null	null	6914.94091796875	null	null	0.0	null	null	null	null	null	single_protein
+PRT	sp|O46375|TTHY_BOVIN	Transthyretin OS=Bos taurus GN=TTR PE=1 SV=1	null	null	18Protein_SoCe_Tr_detergents_trace_target_decoy	null	null	0.105263157894737	sp|O46375|TTHY_BOVIN	null	0.047619047619048	0.0	0.0	0.0	0.0	null	null	0.0	null	null	0.0	null	null	null	null	null	single_protein
```

Comment:
here quant got lost... probably related to how we now store file and label level quants